core/commands/filestore.go

```diff
@@ -201,6 +203,52 @@ var dupsFileStore = &cmds.Command{
 	Type:       RefWrapper{},
 }
 
+var getFileStoreCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "Download IPFS objects into filestore.",
+	},
+	Arguments: []cmds.Argument{
+		cmds.StringArg("cid", true, false, "The cid of the IPFS object to be outputted."),
```

Comment:
maybe s/outputted/fetched/ ?

Reply:
Well, it may not necessary fetch it if its already in local storage.